{{cookiecutter.project_name}}/.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env bash
+poetry install --with dev
+pip install pre-commit
```

Comment:
Wouldn't this be already installed with the `poetry install --with dev` command?

Reply:
Vscode's git integration seems to work just fine with pre-commit now. I've adoped the suggested approach.